backend/src/controllers/course.js

```diff
@@ -20,15 +20,53 @@ exports.getCourse = function ({ params }, res) {
 exports.getCourseQuestions = function ({ params, query }, res) {
     let p = parseInt(query.p)
     const pageNumber = p || 1
-    responseHandler(questionModel.getQuestions(params.code, pageNumber), res)
+    const pageSize = 10
+
+    let getCourseQuestions = new Promise(function(resolve, reject) {
+        Promise.all([
+            questionModel.getQuestions(params.code, pageNumber, pageSize),
```

Comment:
Oh. Shouldn't these be the same query with a nested sub query or does SQLite not allow that?